plugins/qtAccessionPlugin/lib/model/QubitAccessionEvent.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+
+/**
+ * Skeleton subclass for representing a row from the 'accession_event' table.
+ *
+ * 
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements.  This class will only be generated as
+ * long as it does not already exist in the output directory.
+ *
+ * @package    plugins.qtAccessionPlugin.lib.model
+ */
+class QubitAccessionEvent extends BaseAccessionEvent {
+
+  public function getNote()
+  {
+    $criteria = new Criteria;
+
+    $criteria->add(QubitNote::TYPE_ID, QubitTerm::ACCESSION_EVENT_NOTE_ID);
+    $criteria->add(QubitNote::OBJECT_ID, $this->id);
+
+    if ($note = QubitNote::getOne($criteria))
+    {
+      return $note;
+    }
+    else
+    {
+      $note = new QubitNote;
+      $note->objectId = $this->id;
+      $note->typeId = QubitTerm::ACCESSION_EVENT_NOTE_ID;
+
+      return $note;
+    }
```

Comment:
Returning a placeholder note is not something we usually do. Not a big deal but if you need it, you could remove the else block since you're returning in the first condition.

```php
if ($note = QubitNote::getOne($criteria))
{
  return $note;
}

$note = new QubitNote;
$note->objectId = $this->id;
$note->typeId = QubitTerm::ACCESSION_EVENT_NOTE_ID;

return $note;
```